src/peft/tuners/lora/model.py

```diff
@@ -159,18 +160,25 @@ def _create_and_replace(
         target,
         target_name,
         parent,
-        **optionnal_kwargs,
+        **optional_kwargs,
     ):
+        # Regexp matching - Find key which matches current target_name in patterns provided
+        current_key = optional_kwargs["current_key"]
```

Comment:
Since `current_key` is a must have, I would just put it into the normal signature of the method and not inside of `optional_kwargs`.

Reply:
Addressed in the latest commit